Sources/LanguageServerProtocol/Types/Server.swift

```diff
@@ -212,10 +219,17 @@ public class Server {
     public func cursor(forText at: TextDocumentPositionParams) throws -> Hover {
         // If the JSON is "malformed" (e.g., empty object) just return an empty `String`. No errors.
         guard let c = try getCursor(forText: at) else {
-            return Hover(contents: [""], range: .none)
+            return Hover(contents: [], range: .none)
         }
 
-        let contents = [c.annotatedDeclaration, c.fullyAnnotatedDeclaration, c.documentationAsXML].compactMap { $0 }
+        let contents: [MarkedString] = [
+            c.annotatedDeclaration,
+            c.fullyAnnotatedDeclaration,
+            c.documentationAsXML
+        ]
+            .compactMap { $0 }
+            .map { deepString(doc: (try! AEXMLDocument(xml: $0)).root) }
```

Comment:
Can you give an example of what String looks like going into `deepString` and what it looks like coming out of `deepString`?

Also, what are your thoughts on going with `try?` and then using `compactMap`. That would obviously silently fail but it would prevent a hard crash.